frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -354,7 +354,13 @@ export type PrometheusActivityProps = {
   results: PrometheusResponse[];
 };
 
+export type alertType = {
```

Comment:
```suggestion
export type CephHealthCheck = {
```

Reply:
I dont think we need to do the change in common components. The health check can be seperately defined and render in ceph plugin only.